File: Chapter 02/C Programs/2_8.c

#include<stdio.h>
void main()
{
int class_held, class_att;
float Percent_att;
printf("Enter total number of classes held ");
scanf("%d", &class_held);
printf("Enter total number of classes attended ");
scanf("%d", &class_att);

Percent_att = (class_att / (float)(class_held))  *100.0;
if(Percent_att>=80)
{ printf( "Not Debarred");  }
else
{ printf( "You are debarred. "); }

printf("Attendance is %f", Percent_att);
     }

File: Chapter 02/Python programs/2_8.py
class_held = int(input("Total number of classes held: "))
class_att= int(input("Total number of classes attended: "))
Percent_att = (class_att / class_held)  *100
if(Percent_att>=80):
    print( "Not Debarred")
else:
    print( "You are debarred")

print("Your attendance is ",round(Percent_att,2), "%")
